src/anm/player.js

```diff
@@ -377,6 +377,8 @@ Player.prototype.load = function(arg1, arg2, arg3, arg4) {
             state.happens = C.RES_LOADING;
             player.fire(C.S_CHANGE_STATE, C.RES_LOADING);
             player.fire(C.S_RES_LOAD, remotes);
+            // actually start loading remote resources
+            anim._loadRemoteResources(player);
```

Comment:
Hard to say, honestly. If it's not breaking something or fixing something, then let it be. Though, by logic, it seems to me that subscription should be performed _before_ the actual process of loading, so subscriber will get the result even if some resource was received asynchronously faster than the next call of `subscribe` was performed, which is zero-chance and paranoia, yes, but makes feel safer in the order it was before.